src/lib/y2firstboot/clients/licenses.rb

```diff
@@ -69,18 +72,35 @@ def refusal_action
 
       # Directories in which look for the license agreement texts
       #
-      # NOTE: if that result in an empty list, {Yast::InstLicenseClient} will do
-      # an extra attemp to look for the license agreement in the path given as
-      # "base_product_license_directory" global param through the control file.
+      # NOTE: if that result in an empty list and the "directory" argument was ommited too,
+      # {Yast::InstLicenseClient} will do an extra attemp to look for the license agreement in the
+      # path given as "base_product_license_directory" global param through the control file.
+      #
+      # @return [Array<String>] license agreement paths
+      def directories
+        (module_license_directories + sysconfig_licenses_directories).compact.reject(&:empty?).uniq
+      end
+
+      # Directories defined by the module arguments
+      #
+      # NOTE: for the time being, it is only based in the "directory" argument, but it should be
```

Comment:
NP: I would rephrase it a little:

NOTE: right now license module can only define a directory path, but it could be extended to support more than one (e.g., by a "directories" list).

Reply:
> NP: "base in" or "base on" ?

This is a question for me or for @mvidner? ;)

_Based on_ [this answer](https://www.italki.com/question/213376), **based on** is more appropriated.

Thanks!